packages/components/src/DateTimePickers/Date/Input/Input.component.js

```diff
@@ -8,7 +8,10 @@ export default function Input(props) {
 	const { value, inputManagement } = useContext(DateContext);
 
 	return (
-		<InputSizer placeholder={inputManagement.placeholder} inputText={value.textInput}>
+		<InputSizer
+			placeholder={inputManagement.placeholder}
+			inputText={value.date ? value.textInput : ''}
```

Comment:
when input of date is valid, then take the width of input. if date is not valid, take width of placeholder.